index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title> Jon | Johnson </title>
+	<link rel="stylesheet" type="text/css" href="style.css">
+	<script>
+function scrollWin(x, y) {
```

Comment:
Hey Jens,

well.. I initially wanted to do something like if i clicked on the link, it
will scroll down to the portfolio or about me session. I must have forgot
to remove this!

that was before we went through javascript and bootstrap. Probably alot
easier to try now.

On Thu, May 26, 2016 at 10:49 AM, Jens H. Nielsen <notifications@github.com>
wrote:

> In index.html
> <https://github.com/wdi-hk-11/lab-simple-website/pull/3#discussion_r64685144>
> :
>
> > @@ -0,0 +1,63 @@
> > +<!DOCTYPE html>
> > +<html>
> > +<head>
> > +	<title> Jon | Johnson </title>
> > +	<link rel="stylesheet" type="text/css" href="style.css">
> > +	<script>
> > +function scrollWin(x, y) {
>
> what are you using for ? What was your idea ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/wdi-hk-11/lab-simple-website/pull/3/files/868bd57573e349f985a72d784cb728fe968d20f1#r64685144>
>